### ISSUE TITLE
parameters generate command should allow using default values

### ISSUE THREAD

**Issue by carolynvs** [getporter/porter#1604]
When a parameter has a default value defined, we should allow skipping setting that parameter in the parameter set so that they can use the default value. 

Example
```console
$ porter parameters generate hello-llama --reference getporter/hello-llama:v0.1.1
Generating new parameter set hello-llama from bundle hello-llama
==> 2 parameters declared for bundle hello-llama

? How would you like to set parameter "name"
   [Use arrows to move, space to select, type to filter]
  secret
  specific value
  environment variable
  file path
  shell command
> use default value (DEFAULT VALUE)

**Comment by divbhasin** [comment]
I would like to work on this :) Did some digging and I think I know what changes need to be made. Just wanted to clarify that the "default" value for a param refers to the following result of running `porter explain` on `hello-llama`:
 
![Screen Shot 2021-05-24 at 7 52 10 PM](https://user-images.githubusercontent.com/5903786/119420081-a0f82f00-bcc9-11eb-8434-18400be16ab8.png)

**Comment by carolynvs** [comment]
Yup! That is the default value used for the parameter when the user does not specify it. 👍

**Comment by kichristensen** [comment]
Implemented with a skip option in #3100
